iOS automatic linking is not working.

## Summary

Version: 0.17.4 


## Current behavior

RNDeviceInfo automatic linking fails on iOS while it shows success on terminal.
## Expected behavior

RNDeviceInfo should be present in Libraries in Xcode.

I'm a total newbie, but I got the same error above even with the new patched version  `0.21.4`.

```
ld: library not found for -lRNDeviceInfo-tvOS
clang: error: linker command failed with exit code 1 (use -v to see invocation)

** BUILD FAILED **
```

* version:

```
"react-native-device-info": "^0.21.4",
npm                4.6.1
node               8.9.4
react-native-cli   2.0.1
react-native       0.52.0
xcode              9.2
```

* Command

```
- create-react-native-app projectName
- npm run eject
- yarn add react-native-device-info
- react-native link react-native-device-info
- react-native run-ios
```

But I could delete `libRNDeviceInfo-tvOS.a` for the build success.

```
You can manually delete libRNDeviceInfo-tvOS.a in Xcode -> [Your iOS build target] -> Build Phrases -> Link Binary with Libraries. It's a react-native link bug.
```

--- a/README.md
+++ b/README.md
@@ -37,12 +37,13 @@ yarn add react-native-device-info
 ```shell
 react-native link react-native-device-info
 ```
-
 (or using [`rnpm`](https://github.com/rnpm/rnpm) for versions of React Native < 0.27)
 
 ```shell
 rnpm link react-native-device-info
 ```
+*For IOS*
+You still need to run `pod install` after running the above link command inside your `IOS` folder.
 
 ### Manual
 

